ansible/roles/ci-data-repo/templates/freebsd-nsmb.j2

```diff
@@ -0,0 +1,10 @@
+# smbfs configuration
+
+[default]
+workgroup=LIFESCI-AD
+
+[FS0]
+addr=fs0.lifesci.dundee.ac.uk
```

Comment:
Have you thought about making this a variable? It'll be easier if we switch servers in future.